### PR TITLE
fix some bash style issues 

### PR DESCRIPTION
**Pull request by karimamer**
I passed the bash files through shellcheck and modified them to pass through it

### CODE REVIEW

File: runFfmpeg.sh [hunk 1]
```diff
@@ -16,12 +16,12 @@ else
...
 
   # Souce our config
-  source radioFiles/config.sh
+  .radioFiles/config.sh
 
   # Define our private variables
   # KEYINT represents the number of key frames.
   # Should be somewhere between 2 to 4.
-  KEYINT=$(expr $VIDEO_FPS \* 3)
+  KEYINT=${#VIDEO_FPS \* 30}
```
Comment (torch2424) [inline runFfmpeg.sh]: Only thing I can say is this one line. 

I don't remember what I was doing here, but how come you changed to 30 if you don't mind me asking?

Comment (torch2424) [inline runFfmpeg.sh]: Only thing I can say is this one line.

I don't remember what I was doing here, but how come you changed to 30 if you don't mind me asking?

Comment (karimamer) [reply runFfmpeg.sh]: oops that's a mistake. I will fix and push

**Comment by torch2424** [comment]
Thank you soooooo much for cleaning up my mess haha!

I wrote all of this code for fun in about 3 nights (around 2am) while half-asleep, so I knew this was needed haha!

**Comment by karimamer** [comment]
No problem my dude. I need to run this on my pi before we merge into master

File: runFfmpeg.sh (hunk 1 above)
Comment (torch2424) [reply runFfmpeg.sh]: Thanks 👍

**Comment by torch2424** [comment]
Oh cool! Thanks for taking the time, just comment here when it's looking good :)

File: additionalScripts/compileFFmpeg.sh [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 rm -rf ffmpeg-source
 git clone https://git.ffmpeg.org/ffmpeg.git ffmpeg-source
-cd ffmpeg-source
+cd ffmpeg-source || exit
```
Comment (torch2424) [inline additionalScripts/compileFFmpeg.sh]: Thank you so much for this <3

File: additionalScripts/compileFFmpeg.sh [hunk 3]
```diff
@@ -27,13 +27,13 @@ cd ffmpeg-source
...
 --enable-libass --enable-omx --enable-omx-rpi --enable-libfdk-aac \
---prefix=$(pwd)/ffmpeg-build \
+--prefix="$(pwd)"/ffmpeg-build \
 --pkg-config-flags="--static" --libdir=/usr/local/lib --extra-version=ntd_20150128 --disable-shared --enable-static
 make -j 4
 sudo make install
 
 # Multiline comment, this is how ffmpeg output should look
-: `
+: '
```
Comment (torch2424) [inline additionalScripts/compileFFmpeg.sh]: Learned something new today 😄 

https://stackoverflow.com/questions/43158140/way-to-create-multiline-comments-in-bash

File: getFileFromDir.sh [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
+#!/bin/bash
 
 # Function to get random file from a directory
 if [ "$#" -ne 1 ]; then
   echo "getFileFromDir.sh: Bash Script to return a random file from a directory"
   echo " "
   echo "USAGE: ./getFileFromDir.sh [Directory path]"
 else
-  ranfile=$( ls "$1" | sort --random-sort | tail -1 )
+  # Use find instead of ls to better handle non-alphanumeric filenames.
```
Comment (torch2424) [inline getFileFromDir.sh]: Thanks for the comment, didn't know that!

File: runFfmpeg.sh [hunk 5]
```diff
@@ -42,8 +42,8 @@ else
   fix_bounds=true: \
   bordercolor=$FONT_BORDER_COLOR: borderw=1: fontcolor=$FONT_COLOR: \
   textfile=$STREAM_TEXT_PATH: y=$FONT_MUSIC_Y_POS: x=$FONT_MUSIC_X_POS[out]" \
-  -vcodec h264_omx -x264opts keyint=$KEYINT:min-keyint=$KEYINT:scenecut=-1 \
-  -b:v $VIDEO_BIT_RATE -preset veryfast -pix_fmt yuv420p \
-  -c:a libfdk_aac -b:a $AUDIO_BIT_RATE -bufsize 960k \
-  -ar $AUDIO_SAMPLE_RATE -f flv $STREAM_URL
+  -vcodec h264_omx -x264opts keyint="$KEYINT":min-keyint="$KEYINT":scenecut=-1 \
```
Comment (torch2424) [inline runFfmpeg.sh]: Yeah I totally learned the importance of quotes in bash while making this :p

File: startStream.sh [hunk 6]
```diff
@@ -53,7 +53,7 @@ while true ; do
       RANDOM_SONG=""
 
       # Check if we support interludes, and we should show one
-      if [ $RADIO_INTERLUDES = true ] && [ $(expr $(date +%s) % $RADIO_INTERLUDE_INTERVAL_LENGTH) = 0 ]; then
+      if [ "$RADIO_INTERLUDES" = true ] && [ "${#(date +%s) % $RADIO_INTERLUDE_INTERVAL_LENGTH }" = 0 ]; then
```
Comment (torch2424) [inline startStream.sh]: Learned another thing today: http://tldp.org/LDP/abs/html/numerical-constants.html#NUMCONSTANTS

Thanks!

**Comment by torch2424** [comment]
Made [release 1.0](https://github.com/torch2424/piStreamRadio/releases/tag/1.0) just in case, because I don't have time to test this right now :)

File: runFfmpeg.sh (hunk 5 above)
Comment (karimamer) [reply runFfmpeg.sh]: hahahh 👍

File: startStream.sh (hunk 6 above)
Comment (karimamer) [reply startStream.sh]: pretty cool right ?

**Pull request merged**